### ISSUE TITLE
[Feature Request] Add Folder Selection for WallpaperManager

### ISSUE THREAD

**Issue by w745658062** [eythaann/Seelen-UI#355]
Description:
currently, changing wallpapers by selecting images one by one can be quite time-consuming. I would like to suggest adding a new setting mode with two options:

1.The current mode where users select images individually.
2.A new mode that allows users to select an entire folder. The manager would then automatically choose images from this folder at set intervals. Users could also have the option to switch images randomly or based on the file dates (either ascending or descending).

**Additional context**
<img width="1272" alt="20241214" src="https://github.com/user-attachments/assets/f4765de6-e03c-4713-8fe0-23c5166822e6" />

**Comment by PhantomPixel-0418** [comment]
In fact, you can select multiple files in a folder, which you can do with `shift` or `ctrl`

**Comment by eythaann** [comment]
> In fact, you can select multiple files in a folder, which you can do with `shift` or `ctrl`

Yeah, but the feature is more like have a folder where you can just download new images, and they are automatically shown on the slider.

**Comment by w745658062** [comment]
Yes, mainly because I don't want to have to go to WallpaperManager to select the image again every time I download a new one. Actually, I've written a program that fetches images from Bing daily. It would be great if this feature could be added as well.

**Comment by arlandi1974** [comment]
Yes please. In my case, I already have a folder where I put all my wallpapers so whenever I found an image from the net I just save that in that folder and windows just rotate from there. Adding file one by one for the wallpaper manager took too long.
